lib/index.js

```diff
@@ -68,4 +76,14 @@ function hte (client, service, method, datas) {
   }
 }
 
+hte.Request = s.Object({
+  headers: s.Array([ s.String() ]),
+  url: s.String(),
+  method: s.String(),
+  params: s.Object({ strict: false }),
+  query: s.Object({ strict: false })
+}, {
+  strict: false
```

Comment:
Does the `strict: false` here mean that each of the properties of the object are optional?